spring-vault-core/src/main/java/org/springframework/vault/support/VaultDecryptionPayload.java

```diff
@@ -0,0 +1,47 @@
+package org.springframework.vault.support;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Decryption Value Object used for encrypt() operations.
+ * 
+ * @author Praveendra Singh
+ *
+ */
+@Getter
+@Setter
+@AllArgsConstructor
+public class VaultDecryptionPayload {
```

Comment:
@mp911de 
I couldn't find better name and hence selected Payload instead of Request.
Please let me know if you would like to have different naming for the consistency purposes.